test/parser.jl

```diff
@@ -1022,3 +1022,6 @@ facts("misc syntax changes") do
         @fact (Parser.parse(ex) |> norm_ast) --> (Base.parse(ex) |> norm_ast)
     end
 end
+
+# issue #72 : failure to parse and incorrectly specified vector
+Base.@test Parser.parse("[1,2;3]") == :([$(Expr(:parameters, 3));1;2])
```

Comment:
Actually, looking a few lines above this, I think we can follow suit with existing tests and simplify this to
```julia
Base.@test Parser.parse("[1,2;3]") == Base.parse("[1,2;3]")
```
Though since the rest of the tests are using FactCheck, this should probably use the `@fact a --> b` syntax rather than `Base.@test`.